File: Results/DER/Translation/translation_output/misleading_test/gpt-3.5/codenet/python/java/p02952.java
```java
import java.util.Scanner;

public class CountOddLengthNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int ans = 0;
        
        for (int i = 0; i < n; i++) {
            String hoge = String.valueOf(i+1);
            if (hoge.length() % 2 == 1) {
                ans++;
            }
        }
        
        System.out.println(ans);
    }
}
```

File: Results/DER/Translation/translation_output/no_test/codellama/codenet/java/python-sanitized/p02952.py
import java.util.*;

public class p02952 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int a, b, c = 0;
        a = sc.nextInt();
        for (int i = 1; i <= a; i++) {
            if (String.valueOf(i).length() % 2 != 0) {
                c++;
            }
        }
        System.out.println(c);
    }
}



import java.util.Scanner

def main():
    sc = Scanner(System.in)
    a = sc.nextInt()
    c = 0
    for i in range(1, a + 1):
        if len(str(i)) % 2 != 0:
            c += 1
    print(c)

if __name__ == "__main__":
    main()